src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -121,6 +123,10 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this._register(this._configurationService.onDidChangeConfiguration(e => {
 			this.doubleClickEditEnabled = this._configurationService.getValue('notebook.enableDoubleClickEdit');
 		}));
+		this._register(this._configurationService.onDidChangeConfiguration(e => {
```

Comment:
Just add this to the onDidChangeConfiguration above - no reason to have two separate callbacks.